captum/attr/_core/layer/layer_gradient_x_activation.py

```diff
@@ -36,11 +46,17 @@ def __init__(self, forward_func, layer, device_ids=None):
 
     def attribute(
         self,
-        inputs,
-        target=None,
-        additional_forward_args=None,
-        attribute_to_layer_input=False,
-    ):
+        inputs: Union[Tensor, Tuple[Tensor, ...]],
+        target: Optional[
+            Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]]
+        ] = None,
+        method: str = "gausslegendre",
+        internal_batch_size=None,
+        additional_forward_args: Optional[int] = None,
+        attribute_to_layer_input: bool = False,
+    ) -> Union[
+        Tensor, Tuple[Tensor, ...], Tuple[Union[Tensor, Tuple[Tensor, ...]], Tensor]
```

Comment:
This should be just Union[Tensor, Tuple[Tensor, ...]]

Reply:
Fixed